docs/template-creation-how-to-guide.md

```diff
@@ -46,15 +41,15 @@ To get started updating/creating templates:
 
 ## Partial templates
 
-Handlebars allows for nested templates, enabling you to create partial templates to be called by your top level template. If you want to edit a referenced partial template, you can double click on any underlined reference in the template editor and that template will open in a new tab in the bottom left-hand side of the UI. As you make modifications, you will see the results reflected on the right-hand side, which still shows the main top level template you loaded originally.
+Handlebars allows for nested templates, enabling you to create partial templates to be called by your top level template. If you want to edit a referenced partial template, you can double click on any underlined reference in the template editor and that template will open in a new tab . As you make modifications, you will see the results reflected in the FHIR output, which still shows the main top level template you loaded originally.
 
 ![edit partial template](images/partial-template-edit.png)
 
 **NOTE**: When creating a FHIR bundle, it is often required to pull details from several partial templates. When doing this, it is possible that resources could be repeated. To avoid duplicate entries, once the message has been compiled, the FHIR Converter will merge/de-dup any created resources that have the same resourceType, ID, and versionID. An example of a template leveraging this is ADT_A01.hbs.
 
 While you’re modifying a partial template, decide if you want to modify the partial template for all templates that reference this partial template or just the one you’re working on. If you only want to modify it for the one template you are working on, make sure to rename the template at the top before making any changes and hit save. On the main template you will have to update the template to call your newly named partial template.
 
-**NOTE**: Scrolling between the template editor on the left and FHIR bundle output on the right-hand side only works for the original template you selected. When working with a template that you drilled into, you will need to scroll on both sides of the UI.
+**NOTE**: Scrolling between the template editor and the FHIR bundle output only works for the original template you selected. When working with a template that you drilled into, you will need to scroll on both sides of the UI.
```

Comment:
This feature has been removed, the scrolling is no longer synced between the templates and output.

Reply:
Thank you! Removed

Reply:
This whole note should be removed. There is no scroll work done by the UI anymore.